src/main/java/org/isf/dlvrrestype/manager/DeliveryResultTypeBrowserManager.java

```diff
@@ -57,23 +57,23 @@ protected void validateDeliveryResultType(DeliveryResultType deliveryResultType,
 		List<OHExceptionMessage> errors = new ArrayList<>();
 		if (key.isEmpty()) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.dlvrrestype.pleaseinsertacode"),
+					MessageBundle.getMessage("angal.common.pleaseinsertacode.msg"),
 					OHSeverityLevel.ERROR));
 		}
 		if (key.length() > 1) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.dlvrrestype.codetoolongmaxchar"),
+					MessageBundle.getMessage("angal.common.thecodeistoolongmax1char.msg"),
 					OHSeverityLevel.ERROR));
```

Comment:
here too

Reply:
Didn't do this one because of "chars" vs "char".

Reply:
maybe we can change from 1 to "one"